react-crash-course-with-hooks-ca06/notes/01_egghead-how-react-works.md.md

```diff
@@ -0,0 +1,5 @@
+<TimeStamp start="0:22" end="0:31">
+  
+  The ReactDOM library is what is used to actually interact with the DOM
```

Comment:
Since this is a beginner crash course, maybe adding in a note on what the DOM is?

Reply:
Yeah, that sounds good. Also I really need to fix my script 😅